src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+html {
+  --header-nav-color: #00ACDC;
+}
+
+body,
+ul {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  color: #000;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+header>a{
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+}
+
+.logo {
+  width: 100%;
+}
+
+.header_menu {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+  list-style-type: none;
+}
+
+.menu_item {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  text-transform: uppercase;
+  margin-right: 20px;
+}
+
+.menu_item a {
+  position: relative;
+  text-decoration: none;
+  color: inherit;
+  line-height: 60px;
+  transition: color 0.3s ease;
+  padding: 23px 0;
+}
+
+.menu_item a:hover,
```

Comment:
The same:

don't use combined selector and tag selector for specific styles
add class to the element you want to style and use class selector